src/terraform/command_helpers.go

```diff
@@ -148,3 +132,12 @@ func getDependencyServiceEnvVars(deployConfig deploy.Config, serviceConfig types
 	}
 	return result
 }
+func extractDependencyData(dependencyName, dependencyType string, deployConfig deploy.Config) (string, error) {
+	serviceData := deployConfig.AppContext.GetDependencyServiceData(dependencyType)
```

Comment:
I think the input to `GetDependencyServiceData` should be `dependencyName` instead of `dependencyType`